### ISSUE TITLE
How do the buttons work?

### ISSUE THREAD

**Issue by ZahnPastor** [BigE/desk-changer#29]
there are the 4 buttons for controlling the wallpaper, but how do the 2 middle ones exactly work? I.e. When i press the Play-button, the stop button shows up in that place, is it playing now because i pressed play or is it stopped now because i currently shows stop? Same with the shuffle / repeat buttons

**Comment by ZahnPastor** [comment]
ok so through experimenting i found out that it's actually always the option you clicked on last, and the play button seems to do an hourly change, the timer icon seems to use the settings timer and the stop button stops it.
It would be nice if you can make it clear with the buttons what option is actually running now

**Comment by ZahnPastor** [comment]
but wait the random/repeat buttons work like the exact other way around, the one button currently showing is the mode it's in.

fix pls

**Comment by BigE** [comment]
I agree, there are improvements to be made here. I'm personally not a UI person, so any recommendations anyone has on how to make this better, I am open to. I'll leave this open as an improvement to be made once I figure out how to implement the buttons better.

**Comment by ZahnPastor** [comment]
Well the repeat/shuffle is fairly easy to do, most music programs have it as a button, that's grey when the shuffle is off and green/blue when it's on. To show the repeat button is unnecessary in my opinion, because it's a given that an app that changes the desktop wallpaper should repeat changing it and not just stop.

I would make it a faint gray button if the shuffle is off, and display it normal-grey when it's on. [Like this](http://i.imgur.com/H4mCOa8.gif). I don't have any plan on the timer yet, maybe i'll come up with something the following days.

**Comment by ZahnPastor** [comment]
I think for the play buttons you can just keep the current design (the last one you clicked is the option), but remove the timer button, and add it like the option for the daemon. So it's press stop to stop it, press play to play it (standard is hourly change) and then add a slider checkbox that says "use timer instead of hourly change"

**Comment by BigE** [comment]
I've updated the menu controls a bit to help rotation control. As seen in the screenshot, I created a sub menu with the rotation control drop down. I've left the other controls as is for now. I might make a few more tweaks and updates, but this seems like a pretty simple solution.

![screenshot from 2017-08-27 23-06-48](https://user-images.githubusercontent.com/109278/29758852-7ebf4a40-8b7c-11e7-8e9a-c7cc7f363119.png)
